coingecko/interfaces.ts

```diff
@@ -1,12 +0,0 @@
-export type PriceRawResponse = {
-  timestamp: number;
-  price: number;
-}[];
-
-export type Price = {
-  id: number;
-  timestamp: number;
-  price: string;
-};
```

Comment:
got a huuuuuge concern about this setup – having a gazillion keys in our response structure could seriously beef up the payload size. I'm thinking we might wanna explore some alternatives, like maybe using arrays of [id, ts, price]